helix-core/src/main/java/org/apache/helix/task/AbstractTaskDispatcher.java

```diff
@@ -935,6 +933,20 @@ protected void markJobFailed(String jobName, JobContext jobContext, WorkflowConf
         currentTime);
   }
 
+  protected void markPartitionsWithoutLiveInstance(JobContext jobCtx, Set<Integer> allPartitions,
```

Comment:
It seems no one except JobDispatcher needs this method. Can we just move it to there?

Reply:
Sure. Moved to jobDispatcher.